syftlib/src/main/java/org/openmined/syft/execution/Plan.kt

```diff
@@ -16,6 +16,8 @@ private const val TAG = "syft.processes.Plan"
 class Plan(val planId: String) {
     private var pytorchModule: Module? = null
 
+    // The execute function takes in the model parameters, training data and client config as input parameters.
```

Comment:
This is a public api, so I would prefer if you write complete java docs for this class. 

In essence I would like it to have the following:
- description of the class in the beginning
- description of constructor with it's arguments
- what is pytorchModule
- a java doc for execute method
- java doc for generate script

Reply:
Also for these methods describe the output structure. For reference, you can check pytorch java docs